### ISSUE TITLE
App crashes after upgrading to react native 0.75.2

### ISSUE THREAD

**Issue by salisuabubakar** [grahammendick/navigation#804]
Hi, after upgrading my react native to 0.75.2 i get this error after app launch and it crashes as well. Below is my xcode screenshot. Thanks

<img width="1680" alt="Screenshot 2024-08-31 at 11 39 45 PM" src="https://github.com/user-attachments/assets/1c7eb65f-972c-490e-a2e9-4f989d46e240">

**Comment by grahammendick** [comment]
I'm having trouble creating a react native 0.75 project. I'm gonna have to update a few things and try again. 

In the meantime, can you try pasting the following lines [into NVBarButtonComponentView.mm](https://github.com/grahammendick/navigation/blob/56a57dd666b55af0896c11a20af017c4696a7eb9/NavigationReactNative/src/ios/NVBarButtonComponentView.mm#L184), please? You should be able to patch package it?
```obj-c
- (void)didReceiveFailure:(nonnull NSError *)error fromObserver:(nonnull const void *)observer
{
}

- (void)didReceiveProgress:(float)progress loaded:(int64_t)loaded total:(int64_t)total fromObserver:(nonnull const void *)observer
{
}
```

**Comment by grahammendick** [comment]
I finally managed to init a project with react native 0.75. The react native cli defaults to yarn now and yarn wasn't happy with my repo structure 🤷 . The cli reported this as a "pod install failure" so it took me ages to track down.

Anyway, thanks as always for letting me know about the problem. I've just released the fix in [navigation react native v9.22.0 and thanked you in the release notes](https://github.com/grahammendick/navigation/releases/tag/v9.22.0-NavigationReactNative).
